deps/rabbitmq_stream/src/rabbit_stream_utils.erl

```diff
@@ -190,15 +193,50 @@ check_resource_access(User, Resource, Perm, Context) ->
             end
     end.
 
-check_configure_permitted(Resource, User, Context) ->
-    check_resource_access(User, Resource, configure, Context).
+check_configure_permitted(Resource, User) ->
+    check_resource_access(User, Resource, configure, #{}).
 
-check_write_permitted(Resource, User, Context) ->
-    check_resource_access(User, Resource, write, Context).
+check_write_permitted(Resource, User) ->
+    check_resource_access(User, Resource, write, #{}).
 
 check_read_permitted(Resource, User, Context) ->
     check_resource_access(User, Resource, read, Context).
 
+check_super_stream_management_permitted(VirtualHost, SuperStream, Partitions, User) ->
+    Exchange = e(VirtualHost, SuperStream),
+    maybe
+        %% exchange creation
+        ok ?= check_configure_permitted(Exchange, User),
+        %% stream creations
+        ok ?= check_streams_permissions(fun check_configure_permitted/2,
+                                        VirtualHost, Partitions,
+                                        User),
+        %% binding from exchange
+        ok ?= check_read_permitted(Exchange, User, #{}),
+        %% binding to streams
+        ok ?= check_streams_permissions(fun check_write_permitted/2,
+                                        VirtualHost, Partitions,
+                                        User)
+    else
+        _ ->
+            error
+    end.
```

Comment:
Where do we check for `rabbit_vhost_limit:is_over_queue_limit/1`?

Reply:
https://github.com/rabbitmq/rabbitmq-server/issues/9910